gtdynamics.i

```diff
@@ -15,6 +15,18 @@ const string SDF_PATH = kSdfPath;
 const gtsam::KeyFormatter GTDKeyFormatter;
 
 /********************** factors **********************/
+#include <gtdynamics/factors/LinkPoseFactor.h>
+template <JOINT>
```

Comment:
I haven't tested this but doesn't this need to have
`template <JOINT=gtdynamics::RevoluteJoint>` or something?

Maybe if you could make a short python unit test just to check it can import properly?

Reply:
Oooh good call. I think I may have discovered a bug in the wrapper thanks to this.